demosplan/DemosPlanCoreBundle/ResourceTypes/BrandingResourceType.php

```diff
@@ -78,4 +81,47 @@ protected function getAccessConditions(): array
     {
         return [];
     }
+
+    /**
+     * @param Branding $updateTarget
+     */
+    public function getUpdatableProperties(object $updateTarget): array
+    {
+        if ($this->currentUser->hasPermission('area_customer_settings')) {
```

Comment:
Is this correct? Shouldn't it be if !hasPermission?

Reply:
yes, fixed in f88ddc478b522d432117121a758149fc736fefb4